functions/exercise14/script.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+function getAmountOfCoins(amount) {
+    var denominationsOfCoins = [25, 10, 5, 2, 1];
+    var amountOfCoins = [];
+    for (var i = 0; i < denominationsOfCoins.length; i++) {
+        if (amount >= denominationsOfCoins[i]) {
+            amountOfCoins.push(denominationsOfCoins[i]);
+            amount -= denominationsOfCoins[i];
+            i = -1;
```

Comment:
It is not good to change for loop index inside the loop.
Maybe you should consider using while loop here instead of foor loop?